web/packages/teleport/src/UnifiedResources/UnifiedResources.tsx

```diff
@@ -123,11 +132,19 @@ export function ClusterResources({
   const canCreate = teleCtx.storeUser.getTokenAccess().create;
   const [loadClusterError, setLoadClusterError] = useState('');
 
+  const getDefaultIncludedResources = (): IncludedResourceMode => {
+    if (cfg.ui.showResources !== 'requestable') {
```

Comment:
nothing is ever too late, but it would be cumbersome. also, some of this was discussed in the original PR that implemented `showResources` and it was sort of a can of worms that we eventually decided on this. I think i'll leave it for now